tests/fv3_conf/cpld_fv3_ccpp_mom6_cice_cold_atm_flux_run.IN

```diff
@@ -0,0 +1,42 @@
+mkdir INPUT RESTART history MOM6_OUTPUT MOM6_RESTART restart
```

Comment:
May I ask why we need these three new cpld*.IN files? I think the three templates we have cover all the 14 RTs.

Reply:
They were an artifact from merging with the orion branch before it went to the develop branch.  I will delete these.